mrp_production_project_estimated_cost/models/mrp_production.py

```diff
@@ -36,6 +46,17 @@ def _count_created_estimated_cost(self):
         states={'draft': [('readonly', False)]}, default="/")
     created_estimated_cost = fields.Integer(
         compute="_count_created_estimated_cost", string="Estimated Costs")
+    std_cost = fields.Float(string="Estimated Standard Cost")
```

Comment:
Los sumatorios de costes en la OF, efectivamente deben ser readonly. No así en analítica donde el usuario podría cambiar el precio o la cantidad imputada manualmente si hubiese habido algún error.